pkg/types/position.go

```diff
@@ -65,6 +65,17 @@ type Position struct {
 
 	// Modify position callbacks
 	modifyCallbacks []func(baseQty fixedpoint.Value, quoteQty fixedpoint.Value, price fixedpoint.Value)
+
+	// ttl is the ttl to keep in persistence
+	ttl time.Duration
+}
+
+func (s *Position) SetTTL(ttl time.Duration) {
+	s.ttl = ttl
+}
+
+func (s *Position) Expiration() time.Duration {
```

Comment:
do we check the returned Expiration()  in the caller? e.g., skip 0 expiration?

Reply:
I think we don't need to do it. Because if the value is 0, it means it will not expire anymore.